packages/mobile/src/pincode/authentication.test.ts

```diff
@@ -100,6 +100,7 @@ describe(getPasswordSaga, () => {
 describe(getPincode, () => {
   beforeEach(() => {
     jest.clearAllMocks()
+    jest.useRealTimers()
```

Comment:
nit: can we avoid this since we're calling `runAllTimers`? otherwise it seems we're introducing a 800ms delay several times in our test suite.

Reply:
i think my brain is not working, i don't know why i did `useRealTimers` with `runAllTimers`. `runAllTimers` didn't work well, I think because the sleep is inside the tested function. now that i am using the sleep function from the @celo project i just mocked the sleep function and removed these real timers